politeiawww/politeiad.go

```diff
@@ -239,15 +377,124 @@ func (p *politeiawww) getVetted(token, version string) (*pd.Record, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	// Receive reply
 	var gvr pd.GetVettedReply
 	err = json.Unmarshal(resBody, &gvr)
 	if err != nil {
 		return nil, err
 	}
+
+	// Verify challenge
 	err = util.VerifyChallenge(p.cfg.Identity, challenge, gvr.Response)
 	if err != nil {
 		return nil, err
 	}
 
 	return &gvr.Record, nil
 }
+
+// inventoryByStatus retrieves the censorship record tokens filtered by status.
+func (p *politeiawww) inventoryByStatus() (pd.InventoryByStatusReply, error) {
```

Comment:
should we return the data in another format than the whole reply object here?

Reply:
The whole object is fine.

Reply:
Return a pointer though.  Returing pointers from functions lets you do:
`return nil, err`
instead of 
`return pd.InventoryByStatusReply{}, err`

It's much cleaner.